README.md

```diff
@@ -1,2 +1,6 @@
 # tweezet
 A precise, intelligent and multi-use crawler
+
+
+#Decription 
```

Comment:
This is a review where usually someone looks into the changes you have made and suggests improvements, for instance:

```
# Description
```

instead of :
```
#Description
```